django/template/base.py

```diff
@@ -88,6 +88,12 @@
 # than instantiating SimpleLazyObject with _lazy_re_compile().
 tag_re = re.compile(r"({%.*?%}|{{.*?}}|{#.*?#})")
 
+combined_partial_pattern = (
+    r"(\{% \s* partialdef \s+ (?P<name>[\w-]+) (?:\s+ [^%]*)? \s* %\})"
+    r"|(\{% \s* endpartialdef (?:\s+ [^%]*)? \s* %\})"
+)
+combined_partial_re = re.compile(combined_partial_pattern, re.VERBOSE)
```

Comment:
I think we can both simplify the regex and make it more explicit:

```suggestion
combined_partial_re = re.compile(
    r"{%\s*partialdef\s+(?P<name>[\w-]+)(?:\s+inline)?\s*%}"
    r"|{%\s*endpartialdef(?:\s+[\w-]+)?\s*%}"
)
```